apps/frontend/src/components/global/HelpModal.vue

```diff
@@ -70,12 +70,18 @@
 
 <script lang="ts">
 import {AppInfoModule} from '@/store/app_info';
-import Vue from 'vue';
-import Component from 'vue-class-component';
+import ServerMixin from '@/mixins/ServerMixin';
+import Component, {mixins} from 'vue-class-component';
 
 @Component
-export default class HelpModal extends Vue {
```

Comment:
Not sure if this was supposed to be renamed to AboutModal as the HelpModal is used in TopbarDropdown and HelpFooter so these could break.

Reply:
Too funny, done